Headers/Foundation/NSFileManager.h

```diff
@@ -325,6 +325,7 @@ GS_EXPORT_CLASS
 - (BOOL) changeCurrentDirectoryPath: (NSString*)path;
 - (BOOL) changeFileAttributes: (NSDictionary*)attributes
 		       atPath: (NSString*)path;
+- (BOOL) setAttributes:(NSDictionary *)attributes ofItemAtPath:(NSString *)path error:(NSError **)error;
```

Comment:
The spacing here could be more consistent with the rest of the code.

Reply:
In this file we normally have a space after the colon and none before the star (The later is not followed consistently), which actually matches the GNUstep coding style.